.github/workflows/test-container-pr.yml

```diff
@@ -7,6 +7,37 @@ env:
   CONTAINER_NAME: test_container
 
 jobs:
+  lint:
+    name: Lint the code base
+    runs-on: ubuntu-latest
+    steps:
+      - name: Navigate to tests directory
+        run: cd end-to-end-tests/
```

Comment:
We need to initially navigate to `end-to-end-tests/` first, right?

Reply:
Yep! That's correct.

Reply:
Gotcha, I'm trying to figure out why the lint the code base job is failing and it seems like it's around that step: 
`Run cd end-to-end-tests/
/home/runner/work/_temp/23[8](https://github.com/EasyDynamics/oscal-editor-deployment/actions/runs/4854312987/jobs/8651527898?pr=247#step:2:9)5e632-6061-4188-b84c-bca2bc1d9e82.sh: line 1: cd: end-to-end-tests/: No such file or directory
Error: Process completed with exit code 1.`

Reply:
Thanks! That definitely sets me in the right direction,

It seems like the directory change isn't having an effect on the next step though, because it still looks like it's searching for `package-lock` to be in root when attempting to install node: `Error: Dependencies lock file is not found in /home/runner/work/oscal-editor-deployment/oscal-editor-deployment. Supported file patterns: package-lock.json,npm-shrinkwrap.json,yarn.lock`. Is this `actions/setup-node@v3` some type of global install where it's searching at the top-level?